src/main/scala/gemmini/Scratchpad.scala

```diff
@@ -444,9 +445,8 @@ class Scratchpad[T <: Data, U <: Data, V <: Data](config: GemminiArrayConfig[T,
         ex_read_resp.valid := bio.read.resp.valid && !bio.read.resp.bits.fromDMA
         ex_read_resp.bits := bio.read.resp.bits
 
-        val dma_read_pipe = Pipeline(Queue(dma_read_resp), mem_pipeline)
-        val ex_read_pipe = Pipeline(Queue(ex_read_resp), mem_pipeline)
```

Comment:
@jerryz123 Are these queues necessary for functional correctness? Or were they just added to make physical design easier?

Reply:
These don't seem to be in the latest commit on this branch? I think they were not necessary.

Reply:
I removed them myself, but I just wanted to make sure that they weren't necessary